Draft-Accepted/RFCNNNN-Install-PowerShell.md

```diff
@@ -0,0 +1,205 @@
+ not supported while Windows itself is supported.
+
+### Solution
+
+To solve these issues, the proposal is to ship a new command in Windows and the PowerShell Gallery
+that would;
+
+- Be locally discoverable
+- Perform a simple download and install of the latest LTS release of PowerShell
+- Include parameter options to select Stable,Preview or Long Term Service releases
+
+## Goals/Non-Goals
+
+Goals:
+
+- Be locally discoverable, not only an artifact on PSGallery.
+- By default, download and install the latest LTS release to the users CurrentUser Scope
+  - CurrentUser - default installation scope
+  - AllUsers - Administrative users may explicit select this scope
+- Allow users to use remoting for install at scale with Invoke-Command
+- Allow advanced users to specify channel Stable/Preview/LTS
+
+Non-goals:
+
+- Removal of PowerShell (Remove-PowerShell7)
+- Updating of PowerShell (Update-PowerShell7)
+- User selected versions prior to current (No version parameter)
+- Disconnected scenarios - Users that are air-gapped may download the appropriate release from GitHub.
+- Handling of additional MSI switch options - i.e. Enable PSRemoting
+- Cross platform support
+
+## Alternatives
+
+`Install-PowerShell7` cmdlet is an alternative approach that is more common to discover for a
+Powershell user. The cmdlet approach increases the complexity of installing a new version of
+PowerShell from an existing version. Additionally, The ability to bootstrap and install PowerShell
+on a system that currently does not include Powershell is an added benefit for using a command.
+
+## Specification
+
+Command name: Install-PowerShell7
+
+The default behavior is to download and extract the .ZIP package, then install the latest LTS
+release of PowerShell into the user scope allowed by permissions. If the user has administrative
+privilege, they can choose to install to the scope of AllUsers.
+
+- The package install location - $env:LOCALAPPDATA\Microsoft\pwsh
+  - Downloaded packages should be certificate signature verified before installation. 
+  - Failure of the signature should return an error message and stop performing installation.
+- The installer should update the Users PATH for pwsh
+- The installer should enable Microsoft Updates for scope AllUsers. Due to permission requirements,
+  the scope CurrentUser will require manual updating.
+
+The following parameters can be added by experienced PowerShell users to customize the installation.
+
+- Parameter: Scope - overrides the default of CurrentUser and allows the user to specify the
+  installation location AllUsers.
+- Parameter: Channel - options Stable/Preview/LTS - allows the user to select if they want the
+  current preview or Long Term Service (LTS) release. 
+
+## Syntax and Parameter Sets
+
+The syntax of the command is intentionally made similar to that of a cmdlet. Powershell users are
+familiar with a cmdlet syntax and will find this command similar in experience.
+
```

Comment:
Parameter named sets apply to cmdlets not native commands. Is there more than one set? It appears that all parameters are available in the single parameter set. So no need to distinguish between sets when there is only one.

Reply:
thank you @sdwheeler -- with the changes to native command this no longer is needed.